packages/corvid-local-site/src/siteConverter.js

```diff
@@ -199,13 +199,19 @@ const editorDocumentToLocalDocumentFiles = siteDocument => {
 };
 
 const localDocumentFilesToEditorDocument = localDocumentFiles => {
-  const parsedLocalDocumentFiles = localDocumentFiles.map(file =>
-    isPathOfWixFile(file.path)
-      ? Object.assign({}, file, {
+  const parsedLocalDocumentFiles = localDocumentFiles.map(file => {
+    if (isPathOfWixFile(file.path)) {
+      try {
+        return Object.assign({}, file, {
           content: fromWixFileContent(file.content).content
-        })
-      : file
-  );
+        });
+      } catch (_) {
+        throw new Error(`Cannot parse file [${file.path}]`);
```

Comment:
maybe we need to add ParseError and handle it in getSiteDocument?

Reply:
Please elaborate

Reply:
throw new ParseError(file.path) and then catch specific type of error so it will be not some 'fatal-error', but 'parse-error' with specific message